### ISSUE TITLE
You don't seem to have a generator with the name react-native-ignite installed.

### ISSUE THREAD

**Issue by ron-liu** [infinitered/ignite#217]
I followed the guide to start. Firstly, I type 
```
npm install -g react-native-ignite
```
It looks all good. 
and then when I type 
```
ignite new aaaa
```
It complains as below:

```
You don't seem to have a generator with the name react-native-ignite installed.
You can see available generators with npm search yeoman-generator and then install them with npm install [name].
To see the 15 registered generators run yo with the `--help` option.
```

**Comment by GantMan** [comment]
Normally `react-native-ignite` installs `generator-react-native-ignite` for you.  which is the Generator in question.   Go ahead and do `npm i generator-react-native-ignite -g` and please let us know if that solves your issue.

Cheers!

**Comment by ron-liu** [comment]
it doesn't help, I mean installs generator-react-native-ignite.

Now I fixed the issue. It is because I used to have a system node.js installation with an old yo installed too, and later I switched to nvm. For some reason, the new installed yo (located in /Users/{my-name}/.nvm/... )  didn't take effect. "yo" still pointed to the system node.js which is located in /usr/local/... The easiest way to tell is run command "yo -v", in may case, I got "1.1.x" which was very old. 

To solve the problem, just delete /usr/local/bin/yo, and delete /usr/local/lib/node_modules (since I will stick to nvm), and install ignite again, all fixed.
